config.go

```diff
@@ -0,0 +1,94 @@
+package main
+
+import (
+	"os"
+
+	log "github.com/Sirupsen/logrus"
+	flag "github.com/spf13/pflag"
+	config "github.com/spf13/viper"
+)
+
+var (
+	confFile      string
+	logLevel      string
+	restAddress   string
+	rpcAddress    string
+	localStateDir string
+)
+
+var (
+	defaultConfPaths = []string{
+		"/etc/glusterd",
+		".",
+	}
+	defaultConfName = "glusterdconf"
```

Comment:
Purely a personal preference: drop the 'conf' at the end. IOW, files can simply be `glusterd.yaml` or `glusterd.json`